BLOG_POSTING_GUIDELINES.md

```diff
@@ -0,0 +1,61 @@
+# Blog contributing guidelines
+
+Hi! First of all, thank you for wanting to contribute for our blog &#128077;. This document contains all the information you need to know to contribute to our blog page. Should you have any questions, feel free to contact us and we will try our best in answering them.
+
+
+# Before writing
+## Cloning and creating a new branch
+If you have decided to contribute to our blog, the first thing you need to do is clone the repository and creating a new branch where you will be writing our post. In other words, you should run the following commands:
+```
+git clone https://github.com/coderdojobraga/shuriken.git
+cd shuriken
+git checkout -b your-branch-name
```

Comment:
[Since git 2.23](https://github.blog/2019-08-16-highlights-from-git-2-23/) the recommended way to switch between branches is `git switch`. 

```suggestion
git switch -c your-branch-name
```

Read more about this more recent command [here](https://git-scm.com/docs/git-switch/2.23.0).